docker-compose.test.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.2'
+
+## Override for tests in CI
+services:
+    frontend:
+        image: frontend:latest
+    admin:
+        image: admin:latest
+    api:
+        image: api:latest
+    localstack-init:
+        image: localstack-init:latest
+    wkhtmltopdf:
+        image: wkhtmltopdf:latest
```

Comment:
May be good to add ci in the filename for this just to keep it in line with the other ci docker-compose files.

Reply:
Yes good point!